### PR TITLE
Install nvidia gpu driver for GKE cluster as a daemonset.

### PR DESCRIPTION
**Pull request by soumyapani**

### CODE REVIEW

**Review by Chris113113 — APPROVED** [review]

File: aiinfra-cluster/modules/kubernetes-operations/aiinfra-ksa.tf [hunk 1]
```diff
@@ -39,4 +48,14 @@ resource "kubernetes_service_account" "gke-sa" {
       "iam.gke.io/gcp-service-account" = var.node_service_account
     }
   }
+}
+
+// Installing NVIDIA GPU driver daemonset.
+// ref: https://cloud.google.com/kubernetes-engine/docs/how-to/gpus#installing_drivers
+data "http" "nvidia_driver_installer_manifest" {
+  url = "https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded.yaml"
```
Comment (soumyapani) [inline aiinfra-cluster/modules/kubernetes-operations/aiinfra-ksa.tf:56]: @suffiank @sdlin is this the right driver to install ?

Comment (suffiank) [reply aiinfra-cluster/modules/kubernetes-operations/aiinfra-ksa.tf:56]: Any chance we can do 525.105.17? It's the recommended one we're being told at the moment.

**Review by nick-stroud — APPROVED** [review]

File: aiinfra-cluster/modules/kubernetes-operations/aiinfra-ksa.tf (hunk 1 above)
Comment (sdlin) [reply aiinfra-cluster/modules/kubernetes-operations/aiinfra-ksa.tf:68]: From the instructions Grace sent out (https://docs.google.com/document/d/1tCQBzv97KLYxAqCuI88Do38RAef7rLJJu0GL1_QjWTQ/edit#heading=h.kafq2ay54sj2), the daemonset to use is: https://raw.githubusercontent.com/GoogleCloudPlatform/container-engine-accelerators/master/nvidia-driver-installer/cos/daemonset-preloaded-latest.yaml.

I checked the version installed by that, and it installed `525.105.17`.  However it doesn't seem that the version is pinned.

Here is a diff b/w the daemonset you have and the one with "-latest".
https://diff.googleplex.com/#key=E4RmWAXo91ZT

Comment (soumyapani) [reply aiinfra-cluster/modules/kubernetes-operations/aiinfra-ksa.tf]: Updated to use daemonset-preloaded-latest.yaml

**Comment by Str77781** [comment]
Interesting

**Pull request merged**